### PR TITLE
DEV-137 Adds get applicant route

### PR DESCRIPTION
**Pull request by gunnmgirl**
**What kind of change does this PR introduce? (Bugfix, feature, docs update, ...)**

- Feature

**What is the current behavior? (You can also link to an open issue here)**

- https://developish.atlassian.net/browse/DEV-137?atlOrigin=eyJpIjoiYjRlZjBjMGMwZDNmNDMyZDgwZDI4ODBmMjgxMjg3NTUiLCJwIjoiaiJ9

**What is the new behavior (if this is a feature change)?**

- Added get applicant by id route 

**Does this PR introduce a breaking change?**

- No

**Please check if the PR fulfills these requirements**

- [x] The commit message follows our guidelines
- [x] You’ve rebased your work on the current state of the master development tree
- [ ] Tests for the changes have been added (for bug fixes/features)
- [x] Your changes follow to the coding standards
- [ ] Your code is documented

**Other Information**:

### CODE REVIEW

File: src/applicants/applicantsRoutes.js [hunk 1]
```diff
@@ -6,6 +6,7 @@ import applicantsController from "./applicantsController";
 const router = express.Router();
 
 router.get("/all/:page", isAuth, applicantsController.getAllApplicants);
+router.get("/:id", applicantsController.getApplicant);
```
Comment (Sholja) [inline src/applicants/applicantsRoutes.js]: Should this route be protected as well, so that only logged in user can access it?

**Review by Sholja — CHANGES_REQUESTED** [review]
- @gunnmgirl please take a look at my comment, if the route should be used somewhere outside the app, then make sense to not be protected. If that is the case, then the rest is good.

File: src/applicants/applicantsRoutes.js (hunk 1 above)
Comment (gunnmgirl) [reply src/applicants/applicantsRoutes.js]: Added isAuth middleware

**Review by Sholja — APPROVED** [review]
- Good job.

**Pull request merged**